WikiWatershed/model-my-watershed
# Add sign up and forgot password views

To test:

Try logging in with invalid username/password. The forgot link should pop up. Enter email address. Look at the console for the email. It isn't sent for real because we are using develop settings in Django. Follow the activation link. 

Try signing up for a new account. Follow the link in the console.

Try using input that has errors caught on the front-end (not filling required fields) and back-end errors (eg. passwords don't match, username is already in use, etc.).

I've noticed that the analyze view is unable to get the data to make the bar charts. I don't think this commit introduced the error. I might just need to run some secret command to get it to work (I tried destroying the VMs).

See https://github.com/WikiWatershed/model-my-watershed/issues/223

Connects #152
